src/Apps/Auction/Components/PricingTransparency.tsx

```diff
@@ -65,3 +78,53 @@ export const PricingTransparency: React.FC<{}> = () => {
     </Flex>
   )
 }
+
+export const PricingTransparency = withSystemContext(
```

Comment:
I think the pattern used elsewhere is to call this `PricingTransparencyQueryRenderer` and alias it if neccessary at the call site, such as:

```typescript
import { PricingTransparencyQueryRenderer as PricingTransparency } from "Apps/Auction/Components/PricingTransparency"
```

I think that be a bit more explicit than this and the underscore prefixed `_PricingTransparency` naming. What do you think?

Reply:
done: https://github.com/artsy/reaction/pull/3033/commits/ce0c2b3bc2db8027a1099b20c466af794f7e2217